openeo/rest/datacube.py

```diff
@@ -187,9 +193,17 @@ def load_collection(
                     "Unexpected parameterized `spatial_extent` in `load_collection`:"
                     f" expected schema with type 'object' but got {spatial_extent.schema!r}."
                 )
+        elif not spatial_extent or (isinstance(spatial_extent, dict) and spatial_extent.keys() & {"west", "east", "north", "south"}):
+            pass
+        else:
+            valid_geojson_types = [
+                "Polygon", "MultiPolygon", "Feature", "FeatureCollection"
+            ]
+            spatial_extent = _get_geometry_argument(argument=spatial_extent, valid_geojson_types=valid_geojson_types,
+                                                    connection=connection)
```

Comment:
FYI: we try to stay away from this indenting style and ultimately want to converge to the "black" code style (e.g. see https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html)

In the python client (and other projects) we have a [pre-commit](https://pre-commit.com) config to automatically clean up all coding style issues: e.g. see https://open-eo.github.io/openeo-python-client/development.html#pre-commit-for-basic-code-quality-checks for how to set up. I hope it's not too hard to get working on you windows setup